src/encoded/schemas/changelogs/software.md

```diff
@@ -1,4 +1,7 @@
 ## Changelog for software.json
+### Schema version 11
+
+* Added several items to software_type enum; deprecated purpose field but did not remove.
```

Comment:
We should probably just list all the enums here for consistency.

Reply:
do you mean the CHANGES to the enums or just the additions.

Reply:
List just the new additions